docs/0.4.x/en-US/quickstart.md

```diff
@@ -1,25 +1,31 @@
 
-```sh
-cargo install perseus-cli
-```
+2. Install Perseus: After installing Rust, run the following command to install Perseus:
+   ```sh
+   cargo install perseus-cli
+   ```
 
-Now, pop over to some directory where you keep your projects, and run `perseus new my-app`. That will create a new directory called `my-app/`, which you can easily `cd` into, and, once you're there, you can run this command to start your app:
+3. Create a new app: Navigate to your desired projects directory and run `perseus new my-app`. This will create a new directory named `my-app/`. Change into that directory by running `cd my-app`.
 
-```sh
-perseus serve -w
-```
+4. Start the app: Run the following command to start your app:
+   ```sh
+   perseus serve -w
+   ```
 
-The `serve` command tells Perseus to spin up a proper server for your app, rather than just exporting it to a series of static files (doing this lets you use some of the more advanced features of Perseus, if you want), and `-w` tells it to watch the files in the current directory for changes, so that your app will automatically be rebuilt if you change any code. After this command is done, go and take a look at <http://localhost:8080>, and you should see a welcome screen!
+   The `serve` command sets up a server for your app, enabling you to use advanced features of Perseus. The `-w` flag watches for changes in the current directory, automatically rebuilding your app when you modify any code. Once the command is executed, visit <http://localhost:8080> to see the welcome screen!
 
-This command has a few stages. First, there's *Generating your app...*, which will compile your app's *engine-side* (often called the *server-side*, but Perseus has exporting, tinkering, and a million other things that happen there, so we just call the not-browser the *engine* for simplicity) and build all your app's pages. Then, there's *Building your app to Wasm...*, which compiles your app's *browser-side* into WebAssembly, allowing it to be run in the browser. Finally, there's *Building server...*, which just compiles and prepares the server that Perseus will use to run your code. If it's the first time you're running Perseus, there'll also be a stage for *installing external tools...*, in which Perseus downloads some external dependencies, like `wasm-opt`, which helps to supercharge your Wasm in production. Here, Perseus will also use `rustup` to install the `wasm32-unknown-unknown` target, which is Rust's way of saying 'the browser'. If you're not using `rustup`, you'll need to install this target manually.
+   The `perseus serve` command involves several stages:
+   - **Generating your app**: Compiles the engine-side (server-side) of your app and builds all the app's pages.
+   - **Building your app to Wasm**: Compiles the browser-side of your app into WebAssembly (Wasm) for running in the browser.
+   - **Building server**: Compiles and prepares the server used by Perseus to run your code.
```

Comment:
```suggestion
   - **Building server**: Prepares the server used by Perseus to run your code.
```

Strictly not true that this compiles most of the time, that's usually covered by the generation stage (builder and server are in the same program).